aptos-move/framework/aptos-framework/sources/voting.move

```diff
@@ -68,7 +65,7 @@ module aptos_framework::voting {
 
         /// Required. The hash for the execution script module. Only the same exact script module can resolve this
         /// proposal.
-        execution_hash: vector<u8>,
+        execution_hash: address,
```

Comment:
so I guess address for all hashes now?

Reply:
That's a bad hack for public APIs. Prefer `vector<u8>` or String.

Reply:
so hacky

Reply:
I removed my "beautiful" code.